packages/capabilities/src/ucan.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * UCAN core capabilities.
+ */
+
+import { capability, Schema } from '@ucanto/validator'
+import { equalWith, checkLink, and } from './utils.js'
+
+export const UCANLink = Schema.link({
+  version: 1,
+})
+
+/**
+ * Capability can only be delegated (but not invoked) allowing audience to
+ * derived any `store/` prefixed capability for the (memory) space identified
+ * by DID in the `with` field.
+ */
+export const ucan = capability({
+  can: 'ucan/*',
+  with: Schema.did(),
+  derives: equalWith,
+})
+
+/**
+ * `ucan/revoke` capability is a replacement for the
+ * [UCAN Revocation](https://github.com/ucan-wg/spec#66-revocation) that had
+ * been proposed to a UCAN working group and had a tentative support from
+ * members.
+ */
+export const revoke = capability({
+  can: 'ucan/revoke',
+  /**
+   * With MUST be a DID of the UCAN issuer that is in the proof chain of the
+   * delegation been revoked.
```

Comment:
is this always the DID of the issuer of the "context CID" ?

Reply:
It is otherwise anyone could request a revocation. Put it differently that is how we infer context / scope CID, by walking up the chain and finding a proof that corresponds to `with`.